server/datastore/mysql/migrations/tables/20230926082152_AddWindowsMDMTables.go

```diff
@@ -0,0 +1,69 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230926082152, Down_20230926082152)
+}
+
+func Up_20230926082152(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE windows_mdm_commands (
+    -- command uuid is fleet-generated, so that a command has a unique id as
+    -- soon as it is enqueued. It is unique per device, unlike macOS commands
+    -- that may target multiple hosts. Note that a UUID is 36 chars, but for
+    -- consistency with nano_mdm_commands we use 127 here.
+    command_uuid    VARCHAR(127) NOT NULL,
+    device_id       VARCHAR(255) NOT NULL,
+
+    cmd_verb        SMALLINT UNSIGNED NOT NULL,
```

Comment:
Not clear to me why the command verb is an integer? Looks like in the protocol it would be e.g. `Get`, `Replace`, etc. We'll map that to a constant integer?

Also, the ticket mentioned `tinyint(4)`, I took it to mean up to 4 digits (i.e. up to 9999), which is a `smallint`, not a `tinyint`.

Reply:
What do you think @marcosd4h ? After seeing your PR, I think the command verb would be those possible values: https://github.com/fleetdm/fleet/pull/14159/files#diff-48f92386a8e8669440f0ee54c96b1bc848aae1201f3536aa24674f97f36b9a19R892-R903 , so a short string?